packages/accounts-base/accounts_client.js

```diff
@@ -63,6 +66,31 @@ export class AccountsClient extends AccountsCommon {
   }
 
   /**
+   * @summary Register a new login function on client. Intended for OAuth package authors. You can call the login function by using `Accounts.callLoginFunction`.
+   * @locus Client
+   * @param {String} funcName The name of your login function. Used by `Accounts.callLoginFunction`. Should be the OAuth provider name accordingly.
+   * @param {Function} func The actual function you want to call. Just write it in the manner of `loginWithFoo`.
+   */
+  registerClientLoginFunction(funcName, func) {
```

Comment:
What about calling this `registerServiceLogin` (maybe without `Function`?) and having the parameter be `service` (or `serviceName`) since that's what it is?